docs/en/concept/schema-evolution.md

```diff
@@ -0,0 +1,52 @@
+# Schema evolution
```

Comment:
Any info about what is schema evolution? And what's problem it can solve? Which engine support it?